actionpack/lib/action_dispatch/middleware/ssl.rb

```diff
@@ -123,20 +123,12 @@ def flag_cookies_as_secure!(headers)
       end
 
       def redirect_to_https(request)
-        [ @redirect.fetch(:status, redirection_status(request)),
+        [ @redirect.fetch(:status, 308),
```

Comment:
Suggestion (unless using the explicit code number is the convention):
```suggestion
        [ @redirect.fetch(:status, Rack::Utils::SYMBOL_TO_STATUS_CODE[:permanent_redirect]),
```